simcon_project/simcon_project/urls.py

```diff
@@ -32,7 +31,10 @@
     path('redirect-from-login/', RedirectFromLogin, name="RedirectFromLogin"),
     path('student-view/', StudentView, name="StudentView"),
     path('researcher-view/', ResearcherView, name="ResearcherView"),
-    path('settings/', SettingsView, name="SettingsView"),
+    path('user-view/', ResearcherUserView, name="ResearcherUserView"),
+    path('studentuser-view/', StudentUserView, name="StudentUserView"),    
+    path('settings/', ResearcherSettingsView, name="ResearcherSettingsView"),
+    path('student-settings/', StudentSettingsView, name="StudentSettingsView"),
     path('template-start/<str:name>/', TemplateStartView, name="TemplateStartView"),
```

Comment:
We should start using proper casing soon for the view names, but maybe that can be a cleanup ticket later.